### PR TITLE
[6.1.x] SELinux support

### PR DESCRIPTION
**Pull request by a-palchikov**

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -34,7 +34,8 @@
 
 [[override]]
   name = "github.com/mitchellh/go-ps"
-  branch = "master"
+  source = "github.com/gravitational/go-ps"
```
Comment (r0mant) [inline Gopkg.toml]: Do you think it's worth trying submitting the PR upstream? We'll soon fork all our dependencies :)

File: monitoring/port.go [hunk 2]
```diff
@@ -108,10 +107,8 @@ func (c *portChecker) checkProcess(proc process, reporter health.Reporter) bool
 			continue
 		}
 		switch proc.socket.state() {
-		case TimeWait:
-			// ignore sockets in time-wait state since they're going
-			// away soon
-			log.Debugf("Ignoring %v for program %q(pid=%v).", formatSocket(proc.socket), proc.name, proc.pid)
```
Comment (r0mant) [inline monitoring/port.go]: I would keep this log entry.

File: monitoring/procfs_net.go [hunk 3]
```diff
@@ -63,8 +63,11 @@ func (r portCollector) sockets(fetchSockets ...socketGetterFunc) (ret []process,
 
 	for _, socket := range sockets {
 		proc, err := r.findProcessByInode(socket.inode())
-		if err != nil {
-			log.Warn(err.Error())
+		if err != nil && socket.inode() != "0" && !trace.IsNotFound(err) {
```
Comment (r0mant) [inline monitoring/procfs_net.go]: Could you either add a comment explaining what this condition checks for, or move it into a helper method with descriptive name?

File: Gopkg.toml (hunk 1 above)
Comment (a-palchikov) [reply Gopkg.toml]: https://github.com/mitchellh/go-ps/pull/39

**Pull request merged**